spec/defines/vhost_spec.rb

```diff
@@ -1287,6 +1287,27 @@
             ).with_content(%r{## Proxy rules})
           }
         end
+
+        context 'proxy_dest_reverse_match' do
+          let :params do
+            {
+              'docroot' => '/rspec/docroot',
+              'proxy_pass_match' => [
```

Comment:
I just noticed you here test `proxy_dest_match` but you modify the data type of `proxy_dest_reverse_match`.